src/watcher.ts

```diff
@@ -448,22 +448,30 @@ const watchPathsEqual = (
   return true;
 };
 
-const makeWatcher = (
+/**
+ * Exported for testing.
+ *
+ * @param ignoreInitial Ignore the initial "add" events emitted when chokidar
+ * first discovers each file. We already do an initial run, so these events are
+ * just noise that may trigger an unnecessary second run.
+ * https://github.com/paulmillr/chokidar#path-filtering
+ */
+export const makeWatcher = (
   patterns: string[],
   cwd: string,
-  callback: () => void
+  callback: () => void,
```

Comment:
This is unrelated, but what does this callback do?

Reply:
It just gets passed to the `all` callback